data/conf/dovecot/dovecot.conf

```diff
@@ -24,6 +24,11 @@ mail_plugins = </etc/dovecot/mail_plugins
 mail_attachment_fs = crypt:set_prefix=mail_crypt_global:posix:
 mail_attachment_dir = /var/attachments
 mail_attachment_min_size = 128k
+# Significantly speeds up very large mailboxes, but is only safe to enable if
+# you do not manually modify the files in the `cur` directories in
+# mailcowdockerized_vmail-vol-1.
+# https://mailcow.github.io/mailcow-dockerized-docs/u_e-dovecot-perf/
```

Comment:
Please change the docs link to: https://mailcow.github.io/mailcow-dockerized-docs//manual-guides/Dovecot/u_e-dovecot-performance/